release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+a time value in seconds that meets their needs.
+
+There are three possible OAuth client types:
+
+. `openshift-web-console` -  The client used to request tokens for the OpenShift web console.
+
+. `openshift-browser-client` - The client used to request tokens at
+*_/oauth/token/request_* with a user-agent that can handle interactive logins,
+such as using Auth from GitHub, Google Authenticator, and so on.
+
+. `openshift-challenging-client` - The client used to request tokens with a user-agent that can
+   handle WWW-Authenticate challenges, such as the `oc` command line.
+
+- When `accessTokenMaxAgeSeconds` is set to `0`, tokens do not expire.
+- When left blank, {product-title} uses the definition in `master-config`.
+- Edit the client of interest via:
++
+----
+# oc edit oauthclients openshift-browser-client
+----
+
+- Set `accessTokenMaxAgeSeconds` to `600`.
+- Check the setting via:
++
+----
+# oc get oauthaccesstoken
+----
+
+See
+xref:../architecture/additional_concepts/other_api_objects.adoc#accessTokenMaxAgeSeconds[Other
+API Objects] for more information.
+
+[[ocp-37-storage]]
+=== Storage
+
+[[ocp-37-local-persistent-volumes]]
+==== Local Storage Persistent Volumes (Technology Preview)
+
+Local storage persistent volumes is currently in
+xref:ocp-37-technology-preview[Technology Preview].
+
+Local persistent volumes (PVs) now offer the ability to allow tenants to request
+storage that is local to a node through the regular persistent volume claim
+(PVC) process without needing to know the node.  Local storage is commonly used
+in data store applications.
+
+The administrator needs to create the local storage on the nodes, mount them
+under directories, and then manually create the persistent volume (PV).
+Alternatively, they can use an external provisioner and feed it the node
+configuration via `configMaps`.
+
+Example persistent volume named `example-local-pv` that some tenant can now claim:
+
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: example-local-pv
+  annotations:
+    "volume.alpha.kubernetes.io/node-affinity": '{
+      "requiredDuringSchedulingIgnoredDuringExecution": {
+        "nodeSelectorTerms": [
+          { "matchExpressions": [
+            { "key": "kubernetes.io/hostname",
+              "operator": "In",
+              "values": ["my-node"]
+            }
+          ]}
+         ]}
+        }'
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: local-storage
+  local:
+    path: /mnt/disks/vol1
+----
+
+See
+xref:../install_config/configuring_local.adoc#install-config-configuring-local[Configuring
+for Local Volume] and
+xref:../install_config/persistent_storage/persistent_storage_local.adoc#install-config-persistent-storage-persistent-storage-local[Persistent
+Storage Using Local Volume] for more information.
+
+[[ocp-37-tenant-driven-storage-snapshotting]]
```

Comment:
Missing a note (like what we had in 3.6): This feature is currently in Technology Preview and not for production workloads.